File: app/src/main/java/com/railian/maksym/provectustesttask/models/Name.java

package com.railian.maksym.provectustesttask.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Name {

    public Name(String title,String first,String last){
        this.title=title;
        this.first=first;
        this.last=last;
    }

    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("first")
    @Expose
    private String first;
    @SerializedName("last")
    @Expose
    private String last;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getFirst() {
        return first;
    }

    public String getLast() {
        return last;
    }

}

File: app/src/main/java/com/railian/maksym/provectustesttask/models/Name.kt
package com.railian.maksym.provectustesttask.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Name(@field:SerializedName("title")
           @field:Expose
           var title: String?, @field:SerializedName("first")
           @field:Expose
           val first: String, @field:SerializedName("last")
           @field:Expose
           val last: String)
